Mathlib/Analysis/Normed/Lp/WithLp.lean

```diff
@@ -50,8 +51,19 @@ variable (p : ℝ≥0∞) (K : Type uK) (K' : Type uK') (V : Type uV)
 
 namespace WithLp
 
+variable {V} in
+/-- The canonical equivalence between `V` and `WithLp p V`. This should always be used to convert
+back and forth between the representations. -/
+def toLp : V ≃ WithLp p V := .refl _
+
+variable {p V} in
+/-- The canonical equivalence between `WithLp p V` and `V`. This should always be used to convert
+back and forth between the representations. -/
+def ofLp : WithLp p V ≃ V := .refl _
+
 /-- The canonical equivalence between `WithLp p V` and `V`. This should always be used to convert
 back and forth between the representations. -/
+@[deprecated WithLp.ofLp (since := "2024-04-27")]
 protected def equiv : WithLp p V ≃ V := Equiv.refl _
```

Comment:
Making this an `abbrev` (but not an `alias`) for `ofLp` should mean that the new `ofLp` lemmas continue to apply to it